src/com/google/enterprise/adaptor/opentext/OpentextAdaptor.java

```diff
@@ -397,6 +398,21 @@ public void init(AdaptorContext context) {
         this.queryStrings);
     log.log(Level.CONFIG, "opentext.displayUrl.objAction: {0}",
         this.objectActions);
+    // Validate the Display URLs.
+    try {
+      URI uri = getDisplayUrl("", 0L);
```

Comment:
Maybe s/""/"default"/, or for the truly pedantic, queryStrings.keySet().iterator().next()? Or just keep a boolean in the loop below and only call logUnreachableHost once, which I think would be more clear.